swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        inventory = [] if not inventory else inventory
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
+        
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+        self.inventory.remove(my_item)
+        self.inventory += [their_item]
```

Comment:
The `+=` operator when it comes lists functions the same way as `extend`. In a case of adding a single element to a list, I would lean more towards maybe using `append` and not necessarily leveraging the operator to its full extent. More specifically here, I would use the `add` method that our `Vendor` objects have access to.